lib/mollie/method.rb

```diff
@@ -2,7 +2,7 @@ module Mollie
   class Method < Base
     IDEAL             = "ideal"
     CREDITCARD        = "creditcard"
-    MISTERCASH        = "mistercash"
+    BANCONTACT        = "bancontact"
```

Comment:
Can you add the constants `eps` and `giropay`? It's in the list of accepted values for the `method` parameter: https://docs.mollie.com/reference/v2/payments-api/create-payment#parameters